### PR TITLE
feat: bump kube-webhook-certgen to 1.4.3

### PR DESCRIPTION
**Pull request by SandhyaRavi2403**
**What problem does this PR solve?**:
bump kube-webhook-certgen to 1.4.3 with cve fix

`sandhya.ravi@GT9X7CVF5F kommander-applications % trivy i registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.4.3
2024-10-03T00:20:37+05:30	INFO	Vulnerability scanning is enabled
2024-10-03T00:20:37+05:30	INFO	Secret scanning is enabled
2024-10-03T00:20:37+05:30	INFO	If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2024-10-03T00:20:37+05:30	INFO	Please see also https://aquasecurity.github.io/trivy/v0.53/docs/scanner/secret#recommendation for faster secret detection
2024-10-03T00:20:39+05:30	INFO	Detected OS	family="debian" version="12.6"
2024-10-03T00:20:39+05:30	INFO	[debian] Detecting vulnerabilities...	os_version="12" pkg_num=3
2024-10-03T00:20:39+05:30	INFO	Number of language-specific files	num=1
2024-10-03T00:20:39+05:30	INFO	[gobinary] Detecting vulnerabilities...
2024-10-03T00:20:39+05:30	WARN	Using severities from other vendors for some vulnerabilities. Read https://aquasecurity.github.io/trivy/v0.53/docs/scanner/vulnerability#severity-selection for details.

registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.4.3 (debian 12.6)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)`

**Which issue(s) does this PR fix?**:
https://jira.nutanix.com/browse/NCN-102769

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: licenses.d2iq.yaml [hunk 1]
```diff
@@ -579,7 +579,7 @@ resources:
       - url: https://github.com/kubernetes/kubectl
         ref: v0${image_tag#1}
         license_path: LICENSE
-  - container_image: registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20221220-controller-v1.5.1-58-g787ea74b6
+  - container_image: registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.4.3
     sources:
       - url: https://github.com/kubernetes/ingress-nginx
         ref:  controller-v1.5.2
```
Comment (mhrabovcin) [inline licenses.d2iq.yaml:585]: Can you fix this reference and make it dynamic?

File: services/kube-prometheus-stack/63.0.0/defaults/cm.yaml [hunk 2]
```diff
@@ -8,6 +8,9 @@ data:
     ---
     commonLabels:
       prometheus.kommander.d2iq.io/select: "true"
+    patch:
+      enabled: true
```
Comment (mhrabovcin) [inline services/kube-prometheus-stack/63.0.0/defaults/cm.yaml:12]: Why do we need to enable this option?

File: licenses.d2iq.yaml (hunk 1 above)
Comment (SandhyaRavi2403) [reply licenses.d2iq.yaml]: done

File: licenses.d2iq.yaml [hunk 3]
```diff
@@ -579,10 +579,10 @@ resources:
       - url: https://github.com/kubernetes/kubectl
         ref: v0${image_tag#1}
         license_path: LICENSE
-  - container_image: registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20221220-controller-v1.5.1-58-g787ea74b6
+  - container_image: registry.k8s.io/ingress-nginx/kube-webhook-certgen:v1.4.3
     sources:
       - url: https://github.com/kubernetes/ingress-nginx
-        ref:  controller-v1.5.2
+        ref: ${image_tag}
```
Comment (mhrabovcin) [inline licenses.d2iq.yaml:585]: I've checked the repo and the image is versioned in a static file in the repo. We can use the `controller-v1.11.2` ref instead.

Comment (SandhyaRavi2403) [reply licenses.d2iq.yaml]: done

File: services/kube-prometheus-stack/63.0.0/defaults/cm.yaml (hunk 2 above)
Comment (SandhyaRavi2403) [reply services/kube-prometheus-stack/63.0.0/defaults/cm.yaml]: added by mistake

**Review by mhrabovcin — APPROVED** [review]

**Pull request merged**